package.json

```diff
@@ -7,7 +7,9 @@
     "prettier": "prettier \"**/*.{css,html,java,js,ts}\"",
     "eslint": "eslint . --ext ts",
     "swiftlint": "node-swiftlint",
-    "postinstall": "lerna bootstrap"
+    "postinstall": "husky install && lerna bootstrap",
+    "prepublish": "pinst --disable",
+    "postpublish": "pinst --enable"
   },
   "husky": {
```

Comment:
Can this block be removed, then?

Reply:
Yes! You are correct. When I was testing, my `npx husky` was v4 but not that I cleared the cache I can confirm we don't need this anymore